addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.xml

```diff
@@ -20,18 +20,17 @@
                                     <i t-attf-class="fa {{doPrint.status === 'loading' ? 'fa-fw fa-spin fa-circle-o-notch' : 'fa-print'}} me-1" />Print Receipt
                                 </button>
                             </div>
-                            <form t-on-submit.prevent="() => doSendEmail.call()" class="send-email d-flex">
-                                <div class="input-email input-group">
-                                    <input type="email" class="form-control py-3 bg-view" t-attf-placeholder="Email: {{currentOrder.getEmailItems().join(', ')}}" t-model="state.inputEmail" />
-                                    <button class="send btn px-5" t-att-class="isValidEmail() ? 'highlight btn btn-primary' : 'btn-secondary disabled'" type="submit">
-                                        <i t-attf-class="fa {{doSendEmail.status !== 'loading' ? 'fa-paper-plane' : 'fa-fw fa-spin fa-circle-o-notch'}}" aria-hidden="true" />
-                                    </button>
-                                </div>
-                            </form>
+                            <div class="input-group">
+                                <input type="text" class="form-control py-3 bg-view" t-attf-placeholder="Write an email or phone number" t-model="state.input" />
```

Comment:
"Write an email or phone number" is a misleading message when the other modules are not installed. Also, the placeholder should be translated.

Reply:
`t-attf-placeholder="eg.: john.doe@mail.com"` what about this ?